crates/bevy_scene/src/scene_spawner.rs

```diff
@@ -43,12 +43,23 @@ pub struct SceneSpawner {
 
 #[derive(Error, Debug)]
 pub enum SceneSpawnError {
-    #[error("scene contains the unregistered component `{type_name}`. consider adding `#[reflect(Component)]` to your type")]
-    UnregisteredComponent { type_name: String },
-    #[error("scene contains the unregistered resource `{type_name}`. consider adding `#[reflect(Resource)]` to your type")]
-    UnregisteredResource { type_name: String },
-    #[error("scene contains the unregistered type `{type_name}`. consider registering the type using `app.register_type::<T>()`")]
-    UnregisteredType { type_name: String },
+    #[error("scene contains the unregistered component `{type_path}`. consider adding `#[reflect(Component)]` to your type")]
+    UnregisteredComponent { type_path: String },
+    #[error("scene contains the unregistered resource `{type_path}`. consider adding `#[reflect(Resource)]` to your type")]
+    UnregisteredResource { type_path: String },
+    #[error(
+        "scene contains the unregistered type `{std_type_name}`. \
+        consider reflecting it with `#[derive(Reflect)]` \
+        and registering the type using `app.register_type::<T>()`"
+    )]
```

Comment:
Interesting. In what situations would this occur?

Reply:
this error is for when we have a component id which doesn't correspond to a type in the registry. we know it hasn't been `register_type`'d but it also likely it's not even been `Reflect`ed, so i put both suggestions down.

Reply:
How would it be in the registry if it's not `Reflect`?